frontend/public/style/_overrides.scss

```diff
@@ -223,138 +217,33 @@ h6 {
-    --pf-c-nav__list-link--Color: #{$color-pf-black-300};
-    --pf-c-nav__list-link--focus--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
-    --pf-c-nav__list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__list-link--hover--after--BackgroundColor: transparent;
-    --pf-c-nav__list-link--hover--BackgroundColor: #{$color-pf-black-700};
-    --pf-c-nav__list-link--hover--Color: #{$color-pf-white};
-    --pf-c-nav__list-link--m-current--after--BackgroundColor: #{$color-pf-blue-300};
-    --pf-c-nav__list-link--m-current--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__list-link--md--PaddingLeft: 24px;
-    --pf-c-nav__list-link--PaddingBottom: var(--pf-c-nav__list-link--PaddingTop);
-    --pf-c-nav__list-link--PaddingLeft: 16px;
-    --pf-c-nav__list-link--PaddingTop: 17px;
-    // Simple List
-    --pf-c-nav__simple-list-link--active--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--Color: var(--pf-c-nav__list-link--Color);
-    --pf-c-nav__simple-list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--focus--FontWeight: var(--pf-global--FontWeight--normal);
-    --pf-c-nav__simple-list-link--hover--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--hover--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--m-current--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--m-current--Color: var(--pf-c-nav__simple-list-link--active--Color);
-    --pf-c-nav__simple-list-link--PaddingBottom: 2px;
-    --pf-c-nav__simple-list-link--PaddingRight: 5px;
-    --pf-c-nav__simple-list-link--PaddingTop: 2px;
-    // Subnav
-    --pf-c-nav__subnav--MarginTop: 0;
-    --pf-c-nav__subnav--MaxHeight: 100%;
-    // Toggle
-    --pf-c-nav__list-toggle--PaddingRight: 0;
     // Nav Section
     font-size: $co-side-nav-font-size;
-  }
-
-  .pf-c-nav__item .pf-c-nav__simple-list .pf-c-nav__separator {
-    --pf-c-nav__separator--MarginLeft: var(--pf-c-nav__list-link--PaddingLeft);
-    margin-right: var(--pf-c-nav__list-link--PaddingRight);
 
-    @media screen and (min-width: $grid-float-breakpoint) {
-      --pf-c-nav__separator--MarginLeft: var(--pf-c-nav__list-link--md--PaddingLeft);
-      margin-right: var(--pf-c-nav__list-link--md--PaddingRight);
+    .pf-c-nav__simple-list .pf-c-nav__link {
+      font-size: $font-size-base;
     }
-  }
-
-  .pf-c-nav__list {
-    list-style: none; // turn off list-styles to fix bug in Edge
```

Comment:
Did you confirm this is no longer needed for Edge?

Reply:
Should be handled by https://github.com/openshift/console/pull/2790/files#diff-307e5a43a73c45e1f3983abc14bcb686R243